docs/src/main/sphinx/client/cli.rst

```diff
@@ -155,6 +155,9 @@ mode:
   * - ``--editing-mode``
     - Sets key bindings in the CLI to be compatible with VI or
       EMACS editors. Defaults to ``EMACS``.
+  * - ``--extra-headers``
+    - Arbitrary headers to attach to HTTP requests made by the CLI. Separate
```

Comment:
Consider also mentioning what separates names and values, since in HTTP it's a `:`, not `=`, so it's not obvious.

BTW I noticed we don't document other options that use composite values, like `--resource-estimate` or `--extra-credential`.

Reply:
Yeah, I was wondering about that - should I keep it consistent with other composite values in the CLI which use `foo=bar`, or keep it consistent with HTTP headers, and I fell on the side of consistentcy with the CLI.

Reply:
This will definitely need an example with two values.. either in the table or in a separate section after that table that you link to. That could also explain what some use cases are for additional header values..  (like custom auth token maybe?)